client/verta/verta/runtime.py

```diff
@@ -40,11 +38,8 @@ def _delete_thread_logs() -> None:
 
 def _get_validate_flag() -> bool:
     """
-    Return the current thread-local variable for validate or initialize a
-    new boolean.
+    Return the 'validate' attribute of the thread local variable.
     """
-    if not hasattr(_THREAD, 'validate'):
-        _set_validate_flag(False)
```

Comment:
Similar to my comment above: let's keep the check for `_THREAD.validate` as a safeguard.

Reply:
Done